README.md

```diff
@@ -1,8 +1,7 @@
-Extension of Python gettext that lets you use contexts as attributes in
-translation strings.
+Extension of Python gettext that lets you use context translations as
+attributes.
```

Comment:
IMHO previous wording was clearer.
"you use (context (as attributes) [to do] string translations"
the new version leaves me with question: attributes of what? 
I suggest:  "[…] you add context to translated strings, later accessible via attributes.". What do you think?

Reply:
"Context translations" are feature of the gettext itself. This extension "just" lets you access them through attribute. Original gettext method returns you just plain string with translation. This class makes gettext return object which behave as string with translation except you can access context translations by its attributes.

Reply:
this makes sense now :) maybe stick the explanation above into the Readme?